pages/el-otro.js

```diff
@@ -39,13 +45,22 @@ function Home(props) {
 
 Home.getInitialProps = async function(context) {
   let query = context.query.q;
+  // set up multi language support
```

Comment:
@froi maybe we could hide this setup on a lib/util we could reuse. Something like:

```
// ./setup-lunr.js
function setupLunr() {
  stemmerSupport(lunr);
  multi(lunr);
  es(lunr);
  lunr.multiLanguage('en', 'es');
  return lunr;
}

export default setupLunr();
```

```
import lunr from './setup-lunr';
...
```

Reply:
Sure. 

One thing to keep in mind, that setup really depends on the index. If you add another that isn’t multi language that setup is not necessary (English stemmer is used by default) or completely different.

Reply:
Done.